CARET_analyze_cpp_impl/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ find_package(ament_cmake REQUIRED)
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
 
 find_package(pybind11_vendor REQUIRED)
-find_package(pybind11 REQUIRED)
+find_package(pybind11 CONFIG REQUIRED)
```

Comment:
Why did you add the `CONFIG` option?

Reply:
Fixed in https://github.com/tier4/caret_analyze_cpp_impl/pull/198/commits/d2a1afcf18f98b29701904c9e240b24a1f586f69